gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/tasks/CreateManifestTask.java

```diff
@@ -515,28 +259,30 @@ private void validateProjectVersion() {
     }
 
     public static TaskProvider<CreateManifestTask> createManifestTask(Project project, BaseDistributionExtension ext) {
+        Provider<Set<ProductId>> provider = project.provider(
```

Comment:
nit while we iterate on this: Would give this a more descriptive name, e.g. `inRepoProductIds`.

Same for the parameter in: https://github.com/palantir/sls-packaging/blob/cb76b23c0d540da4725d566ab015e52e690e199e/gradle-sls-packaging/src/main/java/com/palantir/gradle/dist/tasks/ResolveProductDependenciesTask.java#L81-L82

Reply:
done